tox.ini

```diff
@@ -1,5 +1,11 @@
 [tox]
-envlist = py36, py37, py38, py39, lint, docs
+envlist = py36, py37, py38, lint, docs
+
+[gh-actions]
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38
```

Comment:
Why not `3.9`?

Move to GH actions LGTM.

Reply:
Currently circleci doesn't cover 3.9. I'd prefer adding 3.9 in a separate PR, this is already complex enough

Reply:
I'm happy to follow up w/ 3.9 support in a separate PR

Reply:
3.9 run here: https://github.com/lpetre/LibCST/actions/runs/1116879480